src/main/java/com/elasticbox/jenkins/model/repository/api/WorkspacesRepositoryApiImpl.java

```diff
@@ -72,6 +72,9 @@ public boolean apply(AbstractWorkspace it) {
         } catch (IOException e) {
             logger.log(Level.SEVERE, "There is no workspaces");
             throw new RepositoryException("There is no workspaces");
+        } catch (NullPointerException npe) {
```

Comment:
Is this becouse client.getWorkspaces()? Could we had a precondition on not nullable client?